src/app.js

```diff
@@ -0,0 +1,166 @@
+const { App } = require('@slack/bolt');
+const JiraApi = require('jira-client');
+
+require('dotenv').config();
+
+const jiraApi = new JiraApi({
+  protocol: 'https',
+  host: process.env.JIRA_HOST,
+  username: process.env.JIRA_USERNAME,
+  password: process.env.JIRA_PASSWORD,
+  apiVersion: 'latest',
+  strictSSL: true
+});
+
+// Initializes your app with your bot token and app token
+const app = new App({
+  token: process.env.SLACK_BOT_TOKEN,
+  signingSecret: process.env.SLACK_SIGNING_SECRET,
+  socketMode: true, // enable the following to use socket mode
+  appToken: process.env.SLACK_APP_TOKEN,
+});
+
+const findIssue = async (issueId) => {
+  try{
+  const issue = await jiraApi.findIssue(issueId);
+  return {
+    key: issueId,
+    summary: issue.fields.summary,
+    status: issue.fields.status.name,
+    priority: issue.fields.priority.name,
+    zendeskTicketCount: issue.fields.customfield_13557 ?? 'No data',
+    url: buildIssueUrl(issueId)
+  };
+  } catch (err) {
+    console.error(`(${issueId}) ${err}`);
+  }
+}
+
+const issueSlackBlock = (issue) => ({
+  "text": `Issue ${issue.key}`,
+  "attachments": [
+    {
+      "blocks": [
+        {
+          "type": "section",
+          "text": {
+            "type": "mrkdwn",
+            "text": `*<${issue.url}|${issue.summary}>*`
+          }
+        },
+        {
+          "type": "section",
+          "fields": [
+            {
+              "type": "mrkdwn",
+              "text": `*Status:*\n${issue.status}`
+            },
+            {
+              "type": "mrkdwn",
+              "text": `*Priority:*\n${issue.priority}`
+            },
+            {
+              "type": "mrkdwn",
+              "text": `*Zendesk Ticket Count:*\n${issue.zendeskTicketCount}`
+            }
+          ]
+        }
+      ]
+    }
+  ]
+});
+
+const buildIssueUrl = (issueId) => {
+  return `${process.env.JIRA_HOST}/browse/${issueId}`;
+}
+// Listens to incoming messages that contain Jira Issue Regex Pattern
```

Comment:
No blank line before function.

```suggestion

// Listens to incoming messages that contain Jira Issue Regex Pattern
```